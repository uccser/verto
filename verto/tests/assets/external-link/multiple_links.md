This is the [first link](https://www.link1.com) and this is the [second link](https://www.link2.com).

This is the [third link](https://www.link3.com).
