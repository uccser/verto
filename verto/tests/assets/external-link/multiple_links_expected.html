<p>This is the <a href="https://www.link1.com" target="_blank">first link</a> and this is the <a href="https://www.link2.com" target="_blank">second link</a>.</p>
<p>This is the <a href="https://www.link3.com" target="_blank">third link</a>.</p>
